docs/backend/openai_api_completions.ipynb

```diff
@@ -220,10 +220,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Structured decoding (JSON, Regex)\n",
-    "You can define a JSON schema or regular expression to constrain the model's output. The model output will be guaranteed to follow the given constraints and this depends on the grammar backend.\n",
+    "## Structured decoding (JSON, Regex, EBNF)\n",
+    "You can define a JSON schema, regular expression or EBNF grammar to constrain the model's output. The model output will be guaranteed to follow the given constraints and this depends on the grammar backend.\n",
     "\n",
-    "SGlang has two backends: [Outlines](https://github.com/dottxt-ai/outlines) (default) and [XGrammar](https://blog.mlc.ai/2024/11/22/achieving-efficient-flexible-portable-structured-generation-with-xgrammar). Xgrammar accelerates JSON decoding performance but does not support regular expressions. To use Xgrammar, add the `--grammar-backend xgrammar` when launching the server:\n",
+    "SGLang has two grammar backends: \n",
+    "- Outlines (default) - `json_schema` & `regex`\n",
```

Comment:
We should add back the link. Can you copy the whole introduction from `docs/references/sampling_params.md:L187`?